Updated the note on event status remapper (#25272)
DataDog/documentation: content/en/service_management/events/pipelines_and_processors/status_remapper.md

* Strings beginning with **o** or **s**, or matching **OK** or **Success** (case-insensitive) map to **OK**
* All others map to **info (6)**

> **Note**: If multiple event status remapper processors are applied to a given event within a pipeline, only the first one in the pipeline’s order is considered. Additionally, across all pipelines that match the event, only the first status remapper encountered (from all applicable pipelines) will be applied.

Reviewer: ```suggestion
**Note**: If multiple event status remapper processors are applied to a given event within a pipeline, only the first one in the pipeline's order is considered. Additionally, for all pipelines that match the event, only the first status remapper encountered (from all applicable pipelines) is applied.
```
